### PR TITLE
react-native-style-queries package

### PR DESCRIPTION
**Pull request by CodingItWrong**
# 📝 Why & how

Adds a new library, `react-native-style-queries`

# ✅ Checklist

- [x] Added library to **`react-native-libraries.json`**

### CODE REVIEW

**Comment by Simek** [comment]
Hello @CodingItWrong, thank you for adding new library to the directory! 👍 

Unfortunately, for the out-of-tree platforms (macOS, tvOS, Win) support we require an example utilizing the relevant platform package, see: https://github.com/react-native-community/directory#%EF%B8%8F-out-of-tree-platforms

To do not block this addition I would suggest to remove the flags, you can always update the entry in the future.

File: react-native-libraries.json [hunk 1]
```diff
@@ -8781,12 +8781,27 @@
...
+  {
     "githubUrl": "https://github.com/pusher/pusher-websocket-react-native",
     "npmPkg": "@pusher/pusher-websocket-react-native",
     "examples": ["https://github.com/pusher/pusher-websocket-react-native/tree/master/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/CodingItWrong/react-native-style-queries",
+    "npmPkg": "react-native-style-queries",
```
Comment (Simek) [inline react-native-libraries.json:8794]:
```suggestion
```

If repository name matches the package name, this field can be skipped.

**Comment by CodingItWrong** [comment]
Thanks @Simek. Requested updates have been applied. I'll resolve conflicts when I have time to sit down at a computer

**Comment by CodingItWrong** [comment]
@Simek conflict is now fixed

**Review by Simek — APPROVED** [review]
Thank you for the updates! 🙏

**Pull request merged**